cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -114,6 +115,26 @@ function ManagementCredentials(props: Partial<StepProps>) {
         AwsOrchestrator.initNodeProfile({ awsState, awsDispatch, errorObject, setErrorObject });
```

Comment:
I may be mistaken but I don't think we need initNodeProfile, or at least we don't need initNodeProfile to store the 'nodeType' map in resources any longer right?

Reply:
Okay, just looked a little closer. I see that you are using the generated nodeType object from RESOURCES when constructing the payload...to populate the controlPlaneNodeType.

I think maybe a slightly cleaner solution would be to modify this to initControlPlaneNodeType so that instead of storing a map in resources we can just do the calculation when the Node Profile value is changed, and then store the controlPlaneNodeType value in the FORM section. I think that would follow the pattern that we are trying to establish more closely.

Reply:
This `initNodeProfile` is used to store the default node type for certain node profile.
So do you mean we are not going to store the map, instead, we just store the single value for the node profile right?
For example, if the user chooses `SINGLE_NODE`, the field will be `t3.large`. But if use switches to `PRODUCTION_READY`, the value will be reset to a new corresponding value.

But if we do not store a map, we need to call some functions to fetch the default value and re-render everytime while use changes its choice.

Reply:
Got it. Gonna modify it in the same way we did yesterday

Reply:
Done modification